jupyter-book/trajectories/rna_velocity.ipynb

```diff
@@ -15,53 +15,117 @@
    ]
```

Comment:
<p>This reads great. I'd move this below a ## Motivation header. I have the impression though that it will overlap greatly with the motivation section of the general TI/pseudotime chapter. It might be worth it to keep most of this text to the TI chapter and to refer to it here (I can tell you how to later then).</p><p> </p><p>It might also be worthwhile mentioning that while TI methods ALWAYS return a result, it might not be biologically plausible. One way of adding more biological prior knowledge for hopefully a more believable (laughs at RNA velo) is to take advantage of additional sources like RNA velocity. </p><p> </p><p>So (feel free to adapt!)</p><ol><li>1 intro sentence</li><li>Ref to TI chapter</li><li>Why RNA velocity -&gt; biological priors, dynamic information, more information sources, [...]</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/multimodal-single-cell-best-practices/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>It's already under in the motivation section so that's fine. And yes, it does overlap with the TI chapter. I'd keep it this way, though. After all, you should be able to read chapters independently of one another, right?</p><p> </p><blockquote>It might also be worthwhile mentioning that while TI methods ALWAYS return a result, it might not be biologically plausible.</blockquote><p>You have the same issue with RNA velocity.</p><p> </p><p> </p><p>Sorry, I'm not sure what exactly you mean when you say "adding biological prior through RNA velocity". Are you talking about modelling splicing dynamics? That'd already be covered by dynamics information so I'd keep it as is (otherwise it is too much content IMO and provide more detail in the follwoing. Could you please elaborate? </p>

Reply:
> It's already under in the motivation section so that's fine. And yes, it does overlap with the TI chapter. I'd keep it this way, though. After all, you should be able to read chapters independently of one another, right?

I guess since this is an online book where people can very deliberately select chapters to read them standalone I agree with you.

> You have the same issue with RNA velocity.
Very true. I wouldn't mind mentioning this twice ^_^ Think novices need to understand this.

> Sorry, I'm not sure what exactly you mean when you say "adding biological prior through RNA velocity". Are you talking about modelling splicing dynamics? That'd already be covered by dynamics information so I'd keep it as is (otherwise it is too much content IMO and provide more detail in the follwoing. Could you please elaborate?

Not the smartest word choices of mine. The only thing that I wanted to say is that we're adding more biological information into our TI model (here RNA velocity) to recover trajectories. You can keep it as is.